src/tmpfiles/tmpfiles.c

```diff
@@ -1329,14 +1341,29 @@ static int path_set_attribute(Item *item, const char *path) {
         return fd_set_attribute(item, fd, path, NULL);
 }
 
+static int write_argument_data(Item *i, int fd, const char *path) {
+        int r;
+
+        assert(i);
+        assert(fd >= 0);
+        assert(path);
+
+        log_debug("Writing to \"%s\".", path);
+
+        r = loop_write(fd, item_binary_argument(i), item_binary_argument_size(i), /* do_poll= */ false);
```

Comment:
Before this change there was an assert on i->argument so NULL was never passed here, now it can happen. Should there be an assert on the return of `item_binary_argument`()?

Reply:
(specifically referring to the calling path from write_one_file)